core/src/blocktree_processor.rs

```diff
@@ -86,15 +90,20 @@ pub fn process_entries(bank: &Bank, entries: &[Entry]) -> Result<()> {
         }
         // else loop on processing the entry
         loop {
+            let mut txs_execution_order : Vec<usize> =  (0..entry.transactions.len()).collect();
```

Comment:
let txs_execution_order = if randomize_tx_order { .. Some(..) } else { None }

Reply:
Considered it. Only issue is, reference to random order vector goes out of scope, in `par_execute_entries`, if I were to pass `Option` instead of an array in mt_group.